app/views/steps/dwp/confirm_result/edit.html.erb

```diff
@@ -19,7 +19,8 @@
   <div class="govuk-grid-column-two-thirds">
     <%= step_form @form_object do |f| %>
       <%= f.govuk_collection_radio_buttons :confirm_result, @form_object.choices, :value,
-                                           inline: false %>
+                                           inline: false,
+                                           legend: { aria_describedby: 'dwp-confirm-result-desc' }%>
```

Comment:
I think it is a dash?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

Reply:
Good spot - fixed